grakn-engine/src/main/java/ai/grakn/engine/controller/response/MetaConcept.java

```diff
@@ -46,9 +44,9 @@ public static MetaConcept create(
             @JsonProperty("id") ConceptId id,
             @JsonProperty("@id") Link selfLink,
             @JsonProperty("label") Label label,
-            @JsonProperty("super") @Nullable Link sup,
-            @JsonProperty("subs") Set<Link> subs
+            @JsonProperty("super") @Nullable EmbeddedSchemaConcept sup,
+            @JsonProperty("subs") Link subs
     ){
-        return new AutoValue_MetaConcept(Schema.BaseType.TYPE.name(), id, selfLink, label, false, sup, subs, true, Collections.emptySet(), Collections.emptySet(), Collections.emptySet());
+        return new AutoValue_MetaConcept(Schema.BaseType.TYPE.name(), id, selfLink, label, false, sup, subs, true, null, null, null, null);
```

Comment:
These could be regular links to empty collections if we want. That feels like a bit less of a special-case to me.

Reply:
Ya fair enough. Less nulls running around.